### ISSUE TITLE
Can't use Swift Package Manager as described in README

### ISSUE THREAD

**Issue by jeremybower** [kylef/JSONWebToken.swift#27]
Using:

```swift
import PackageDescription

let package = Package(
  name: "AwesomeProject",
  dependencies: [
    .Package(url: "https://github.com/kylef/JSONWebToken.swift", versions: Version(1,4,2)..<Version(1,5,0))
  ]
)
```

Results in:

```
Downloading dependencies...
Cloning https://github.com/kylef/JSONWebToken.swift
error: The dependency graph could not be satisfied. The package (https://github.com/kylef/JSONWebToken.swift) with version tag in range (1.4.2..<1.5.0) is not found. Found tags ([1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.3.0, 1.4.0, 1.4.1])
```

The repo has no tag for 1.4.2.

**Comment by kylef** [comment]
Sorry, will make a tag tomorrow. Wanted to get another change in before releasing and didn't get round to it yet.

**Comment by kylef** [comment]
Released JSONWebToken 1.4.3 which includes Swift Package Manager support.
